Exercism/Gigasecond/Gigasecond.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Exercism.Gigasecond
+{
+    public static class Gigasecond
+    {
+        public static DateTime addGigasecond(DateTime birthDate) => birthDate.AddSeconds(1000000000);
```

Comment:
You're right I don't know what I was thinking.

I feel like I should be passing some of these on to Exercism.